templates/web/guides/misc/callbacks.md.erb

```diff
@@ -266,6 +267,15 @@ guaranteed to be called even if the thread crashes.
 An example use cases is to ensure that any temporary files are deleted.
 
 
+<a class="anchor" name="mode_changed"></a>
+
+### Mode Changed
+
+#### on_mode_changed
```

Comment:
Should reflect that this takes an options hash.

Should also note here that the new mode will be passed in via options[:mode].